app/src/main/res/values/settings.xml

```diff
@@ -105,6 +106,14 @@
         <item>tags</item>
     </string-array>
 
+    <string-array name="settings_values_auto_backup_password_enc" translatable="false">
+        <item>OFF</item>
+        <item>NEW_ENTRIES</item>
+        <item>ALL_EDITS</item>
```

Comment:
We use lowercase values for everything else so I think we should keep using them here as well.
With that you only would need to use toUpperCase in the SettingsActivity since you already use toLowerCase in the Settings helper class.